nix/mobile/default.nix

```diff
@@ -9,19 +9,52 @@ let
     version = "10.2.1";
   };
   xcodeWrapper = composeXcodeWrapper xcodewrapperArgs;
-  androidPlatform = callPackage ./android.nix { inherit config gradle; };
-  selectedSources =
-    [ status-go ] ++
-    lib.optional platform.targetAndroid androidPlatform;
+  androidPlatform = callPackage ./android { inherit config pkgs target-os mkFilter nodejs maven localMavenRepoBuilder projectNodePackage prod-build-fn; status-go = status-go.android; };
+  iosPlatform = callPackage ./ios { inherit config pkgs mkFilter xcodeWrapper projectNodePackage; status-go = status-go.ios; };
+  fastlane = callPackage ./fastlane { inherit stdenv target-os; };
+  selectedSources = [
+      fastlane
+      status-go.android
+      status-go.ios
+    ] ++
+    lib.optional platform.targetAndroid androidPlatform ++
+    lib.optional platform.targetIOS iosPlatform;
+
+  yarn2nix = import (fetchFromGitHub {
```

Comment:
I really think stuff like that should be its own separate file. It makes it much harder to read what is going on here.

Reply:
Agree.